portfolio/src/main/webapp/index.html

```diff
@@ -1,42 +1,52 @@
-        <a href="#Contact" class="button" id="contact">Contact</button>
-    </div>
-    </div>
-    <div class="size font textsize fontcolor left" id="about">
-     <div id ="bio">
-    <h2 id="About">About me!</h2>
-     <p class="wordtext">My name is <b>Bless Paynes</b> and I'm a junior currently attending <b>Hampton University</b>. I've lived in <b>Northern VA </b> my entire life but I have travelled a ton in the States!</p>
-     <p class="wordtext">Some of my many hobbies include making clothes, knitting, crocheting, drawing and I also run a small but ever growing nail business.</p> 
-     </div>
-     <img id="selfie" src="/images/selfie.jpeg" alt="me smiling" width="400" hieght="400" class="right">
-    </div>
-    <div class="size font textsize center fontcolor" id="experience">
-    <h3 id="Experience">Experience</h3>
-      <ul class="wordtext" id="list">
-        <li>2x Google STEP (Previously known as EP) Intern</li>
-        <li>Worked on Android App team</li>
-        <li>Worked on Google Fi team</li>
-        <li>Rock Paper Scisscor game</li>
-        <li>Utilizing GRPC</li>
-      </ul>            
-    </div>
-    <div class="size font textsize center fontcolor" id="contact">
-     <h2 id="Contact">Contact Me</h2> 
-     <p style="color:purple;">Bpaynes4291@gmail.com</p> 
-     <a href="https://github.com/BlessP/Step"><p style="color:purple;">Github</p></a>
+        <a href="#Experience" class="button" id="experience">Experience</a>
+        <a href="#Contact" class="button" id="contact">Contact</a>
+      </div>
+      <div class="size font textsize fontcolor left" id="about">
+        <div id="bio">
+          <h2 id="About">About me!</h2>
+          <p class="wordtext">
+            My name is <b>Bless Paynes</b> and I'm a junior currently attending <b>Hampton University</b>. I've lived in
+            <b>Northern VA </b> my entire life but I have travelled a ton in the States!
+          </p>
+          <p class="wordtext">
+            Some of my many hobbies include making clothes, knitting, crocheting, drawing and I also run a small but
+            ever growing nail business.
+          </p>
+        </div>
+        <img id="selfie" src="/images/selfie.jpeg" alt="me smiling" width="400" hieght="400" class="right" />
+      </div>
+      <div class="size font textsize center fontcolor" id="experience">
+        <h3 id="Experience">Experience</h3>
+        <ul class="wordtext" id="list">
+          <li>2x Google STEP (Previously known as EP) Intern</li>
+          <li>Worked on Android App team</li>
+          <li>Worked on Google Fi team</li>
+          <li>Rock Paper Scisscor game</li>
+          <li>Utilizing GRPC</li>
+        </ul>
+      </div>
+      <div class="size font textsize center fontcolor" id="contact">
+        <h2 id="Contact">Contact Me</h2>
+        <p style="color: purple;">Bpaynes4291@gmail.com</p>
+        <a href="https://github.com/BlessP/Step"><p style="color: purple;">Github</p></a>
```

Comment:
Same here. You can probably use the same `class` for both since they have the same style.

Note that I would suggest to put the `<p>` before the `<a>`, as it's a more common practice (the link is part of the paragraph, rather than the paragraph being part of the link)

```html
<p class="purple_text"><a href="your-link-here">Github></a></p>
```